tests/test_installed_hermes_session_contract.py

```
@@ -0,0 +1,348 @@
+        self,
+        *,
+        python: Path,
+        cwd: Path,
+        env: dict[str, str],
+        stderr_path: Path,
+    ) -> None:
+        self._next_id = 1
+        self._stderr_path = stderr_path
+        self._stderr = stderr_path.open("w", encoding="utf-8")
+        self._messages: queue.Queue[dict[str, object] | None] = queue.Queue()
+        self._process = subprocess.Popen(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)